core/jreleaser-utils/src/main/java/org/jreleaser/util/signing/Keyring.java

```diff
@@ -109,12 +110,30 @@ public PGPPublicKey readPublicKey() throws SigningException {
             Iterator<PGPPublicKey> keyIter = keyRing.getPublicKeys();
             while (keyIter.hasNext()) {
                 PGPPublicKey key = keyIter.next();
-                if (key.isEncryptionKey()) {
+                if (isSigningKey(key)) {
                     return key;
                 }
             }
         }
 
         throw new SigningException("Did not find public key for signing.");
     }
+
+    /**
+     * Returns {@code true} if the given key can be used for signing.
+     *
+     * <p>There is no Method key.isSigningKey(), and encryption does not always mean signing.
+     * The algorithms here need to be kept in sync with {@code org.bouncycastle.openpgp.operator.jcajce.OperatorHelper#createSignature}.
+     *
+     * @param key they key to check if it is usable for signing.
+     * @return {@code true} if the given key can be used for signing.
+     */
+    private static boolean isSigningKey(PGPPublicKey key) {
+        final int algorithm = key.getAlgorithm();
+
+        return algorithm == PublicKeyAlgorithmTags.EDDSA || algorithm == PublicKeyAlgorithmTags.ECDSA
```

Comment:
Thanks! May I suggest formatting with 1 algorithm per line and the `||` operator at the end of each line?

Reply:
Sure!